### DISCUSSION TITLE
Easiest way to backup updated dotfiles to remote repo

### DISCUSSION THREAD

**Discussion by spa5k** [twpayne/chezmoi#2308] [Q&A] [answered]
So let's say - 
1. I updated by `.zshrc` and few other zsh related files.
2. Now i need to manually run `chezmoi add .zshrc` and same for other files since they are in different folders.
3. Now i need to `chezmoi cd`
4. Then `git add .`, `git commit -m "update"` and then `git push`.

This is pretty long, also one might forget which files were already in the remote repo.

Is/Can there a single command like `chezmoi update` which updates the tracked file, then push them to the repo. or at least update the tracked files, and git stuff can be aliased that is done after `chezmoi cd`

**Comment by twpayne** [comment, marked as answer]
First, configure [automatic committing and pushing](https://www.chezmoi.io/user-guide/daily-operations/#automatically-commit-and-push-changes-to-your-repo) then use the [`re-add` command](https://www.chezmoi.io/reference/commands/re-add/).

**Comment by spa5k** [comment]
Thanks @twpayne it worked like a charm!
